packages/domains-table/src/utils/resolve-domain-status.tsx

```diff
@@ -1,4 +1,10 @@
 import { localizeUrl } from '@automattic/i18n-utils';
+import {
+	SETTING_PRIMARY_DOMAIN,
+	INCOMING_DOMAIN_TRANSFER_STATUSES_IN_PROGRESS,
+	GDPR_POLICIES,
+	DOMAIN_EXPIRATION,
+} from '@automattic/urls';
```

Comment:
Do we need to update the dependencies in `domains-table/package.json` for this?

Reply:
Thanks. Odd that this didn't trigger a build error 🤔 

Updated: [cc66333](https://github.com/Automattic/wp-calypso/pull/88281/commits/cc66333897b0fdbaab426e75271ca8be201a775f)